src/_blog/SRI-roadmap-2025.md

```diff
@@ -0,0 +1,71 @@
+
+This means our team needs to be in sync, adapting to this new situation and plan out how to tackle these efforts nearly in parallel. It’s suggested that we don’t start working on roles until a new testing suite is in place. Once that is done, we should work with early adopters, getting as much feedback as we can on roles, encouraging them to contribute back to our codebase, as we work on protocols.
+
+Below, is a high-level look at our roadmap for the next few months. This plan focuses on improving the maturity of our codebase through protocol refactoring, specification stabilization, and removing technical debt, all while keeping early adopters engaged and supported throughout the process, aiming to initiate work on roles,  always having the same mission: SRI adoption. It it's a dynamic plan, subject to change depending on early adopter interactions.
+
+Higher-resolution image available [here](https://drive.google.com/file/d/1CWFt9o7NOFYfdJrgashLlsAagz7PBZxO/view?usp=sharing) | [Figma File](https://www.figma.com/design/JOgHHrAeRYLHjn6Hc2Hfle/SRI-Roadmap-2024%2F25?node-id=0-1&t=woeUq9LRsGTiT3Ly-1)
+
+![Roadmap ](/assets//SRI-Roadmap-2024.png)
+
+## October 
+- SRI **1.1.0 launch.** This release includes the base for comprehensive protocol documentation, and addition of the integration tests framework.
+- We are establishing regular **communication channels and dedicated support** for all potential adopters and the core team. This involves organizing calls and support channels to ensure a smooth adoption process.
+- A dedicated [auditor role](https://github.com/stratum-mining/stratum/discussions/1052) working group is being established to explore the specifics of the auditor role. This role will focus on transparency and accountability of shares, addressing the need for better auditing processes in mining operations.
+- Cleaning up the **Mining Protocol Specs** [#98](https://github.com/stratum-mining/sv2-spec/pull/98)
+
+## December
+
+- Add variable difficulty feature to the pool
+- Launch SRI 1.2.0 - ensures protocol crates are all well documented and finalized and integration tests framework together with tests is in place, basically unifying the efforts that started in the past 2-3 months.
+- Begin estimation on refactoring and improvements for protocols
+- Continue our Auditor Role research alongside the development efforts. 
+- Benchmarking Case Study launched by an adopter, showcasing the advantages of SV2 in a real-world scenario. This case study will serve as a proof point to illustrate SV2’s benefits to other potential adopters.
+-Encouraging early adopters to contribute back to SRI to avoid code fragmentation and ensure easier maintenance long-term (these contributions will most likely be in roles)
```

Comment:
```suggestion
- Encouraging early adopters to contribute back to SRI to avoid code fragmentation and ensure easier maintenance long-term (these contributions will most likely be in roles)
```

Reply:
What do you mean by fragmented, implementors doing their own roles? Unfortunately that doesn't seem to be technical reality at least for now, as mentioned in the doc adopters want to build quicker, and roles get them quicker towards their goal. Well at least for now.